docs/source/_topic_map.yml

```diff
@@ -5,7 +5,7 @@ Distros: openshift-*
 Topics:
   - Name: Overview
     File: index
-  - Name: Getting Started
+  - Name: Getting started
```

Comment:
it's not consistent there either but you're right, now I see that for the most part they capitalize so I'll revert